model/src/test/kotlin/PackageReferenceTest.kt

```diff
@@ -64,5 +65,17 @@ class PackageReferenceTest : WordSpec() {
                 }
             }
         }
+
+        "findReferences" should {
+            "find references to an existing id" {
+                root.findReferences(Identifier.fromString("::node1_1_1")) shouldBe listOf(node1_1_1)
+                root.findReferences(Identifier.fromString("::node1")) shouldBe listOf(node1)
```

Comment:
Would be nice to have a test where `findReferences` returns more than one value to verify that this works correctly.

Reply:
Agreed, but then I'd need to extend the current node structure and also adjust the existing test accordingly. I'll see what I can do.

Reply:
See https://github.com/heremaps/oss-review-toolkit/pull/1070.